indexer-app/src/cmr/indexer/data/concepts/collection/community_usage_metrics.clj

```diff
@@ -5,9 +5,18 @@
     [cmr.indexer.data.metrics-fetcher :as metrics-fetcher]))
 
 (def not-provided-version
-  "In EMS community usage CSV, the version value when the version is unknown"
+  "DEPRECATED
+   In EMS community usage CSV, the version value when the version is unknown"
   "N/A")
 
+(defn- coll-version-matches-metrics-version?
+  "Returns whether the collection version matches the community usage metrics version.
+   This also returns true if the metrics version is nil or N/A"
+  [parsed-version version]
+  (or (= parsed-version (util/parse-version-id version))
+      (= not-provided-version version)
```

Comment:
what does it mean to be deprecated if you are still using it in new code? What is the project convention here with relation to line 8?

Reply:
Marked as deprecated since it's not in the documentation for updating community usage metrics that that is what the value is supposed to be. But it's still in use by tests and possibly used by other parties. It may be removed entirely eventually.